protos/xyz/block/ftl/v1/console/console.proto

```diff
@@ -50,6 +50,7 @@ message LogEntry {
 message Verb {
   schema.Verb verb = 1;
   string schema = 2;
+  string json_request_schema = 3;
```

Comment:
Does every Verb have this, or just the one we're operating on? This could get very large very quickly if it's a schema for every verb.

Reply:
Yeah good call out. Right now every verb has this. We might want to optimize the calls at some point so we make a request when we get to this form page. But for now, yeah they all will have the jsonSchema, ftlSchema and the stuff that is part of the `schema.Verb`